SDWebImage/Core/SDWebImageManager.m

```diff
@@ -96,20 +96,7 @@ - (nonnull instancetype)initWithCache:(nonnull id<SDImageCache>)cache loader:(no
 }
 
 - (nullable NSString *)cacheKeyForURL:(nullable NSURL *)url {
-    if (!url) {
-        return @"";
-    }
-    
-    NSString *key;
-    // Cache Key Filter
-    id<SDWebImageCacheKeyFilter> cacheKeyFilter = self.cacheKeyFilter;
-    if (cacheKeyFilter) {
-        key = [cacheKeyFilter cacheKeyForURL:url];
-    } else {
-        key = url.absoluteString;
-    }
-    
-    return key;
+  return [self cacheKeyForURL:url context:nil];
```

Comment:
This two API does not do the same thing. They are not just `default nil param`. Don't change this line :)